torchvision/ops/feature_pyramid_network.py

```diff
@@ -80,19 +107,23 @@ def forward(self, x):
         names = list(x.keys())
         x = list(x.values())
 
-        last_inner = self.inner_blocks[-1](x[-1])
+        last_inner = self.last_inner_block(x[-1])
         results = []
-        results.append(self.layer_blocks[-1](last_inner))
-        for feature, inner_block, layer_block in zip(
```

Comment:
I was wondering if it was possible to remove the `zip` and just iterate over the indices?
Something like
```python
for i in range(n, -1, -1):
    feature = x[i]
    inner_block = self.inner_blocks[i]
    ...
```
This would allow to keep backwards-compatibility with the current modules maybe?

Reply:
Blocked by https://github.com/pytorch/pytorch/issues/16123